### ISSUE TITLE
fontawesome5 support?

### ISSUE THREAD

**Issue by Freed-Wu** [billryan/resume#54]
perhaps update fontawesome.sty to [fontawesome5](https://github.com/JanHendrikDolling/latex-fontawesome5)? the fontawesome has a bit obsolete.

**Comment by billryan** [comment]
Yep, I tried fontawesom5 in my local env, but online platform such as overleaf did not support fontawesome5. I will check whether Overleaf support fontawesome5 1 May holiday again.

**Comment by cuhkjpan** [comment]
Hi, can you tell me how to use fontawesom5? Thank you in advance.
